backend/app/api/endpoints/web_scraper.py

```
@@ -0,0 +1,72 @@
+from app.schemas.knowledge import WebScrapeRequest, WebScrapeResponse
+from app.services.web_scraper import get_web_scraper_service
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.post("/scrape", response_model=WebScrapeResponse)
+async def scrape_web_page(
+    request: WebScrapeRequest,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)